projects/plugins/jetpack/composer.json

```diff
@@ -59,6 +59,7 @@
 		"automattic/jetpack-changelogger": "@dev",
 		"automattic/patchwork-redefine-exit": "@dev",
 		"automattic/phpunit-select-config": "@dev",
+		"automattic/wordbless": "^0.5.0",
```

Comment:
If WorDBless is needed (most existing Jetpack tests rely on having an actuall DB-full WordPress install to run against), you should use `automattic/jetpack-test-environment` instead. See #41057.